package.json

```diff
@@ -65,6 +65,7 @@
         "firebase": "^5.9.4",
         "html-react-parser": "^0.7.0",
         "image-webpack-loader": "^4.6.0",
+        "javascript-time-ago": "^2.0.1",
```

Comment:
Why are you adding this package?

Reply:
Hey Roy, it's about time formatting in js but I think its included when you install react-time-ago, Why need to install it again solely @joelethan ?.

Reply:
@wasibani-roy this helps in time formatting to have the since the comment was made